webanno/webanno
# #1207 - Allow disabling validation

**What's in the PR**
- Add option to control if validation happens or not
- Export/import the new setting
- Default to no validation for existing or imported projects where this setting did not exist yet
- By default enable validation for new layers/projects but disable validation for the Token layer
- Update documentation text and screenshot

**How to test manually**

* Validation
  * Change default settings for dependency relations to allow cross-sentence
  * Create a cross-sentence dependency relation
  * Disable cross-sentence for dependency relations again
  * Ensure that the validation setting is on *never*
  * Mark document as finished
  * Set document back to in-progress using the monitoring page
  * Ensure that the validation setting is on *always*
  * Mark document as finished

* Import/export
  * Import old project and check that validation is *never*
  * Change validation to *always* for some layer
  * Export this project again
  * Re-import
  * Check that the validation on the layer is still set to *always*

* New layers
  * Create a new layer
  * Check that the validation is *always*

**Automatic testing**

**Documentation**
